tests/test_core.py

```diff
@@ -1219,7 +1219,11 @@ def test_exceptions_minimal_runtime(self):
       self.do_run_from_file(path_from_root('tests', 'core', 'test_exceptions.cpp'), path_from_root('tests', 'core', 'test_exceptions_caught.out'))
 
       self.set_setting('DISABLE_EXCEPTION_CATCHING', 1)
-      self.do_run_from_file(path_from_root('tests', 'core', 'test_exceptions.cpp'), path_from_root('tests', 'core', 'test_exceptions_uncaught.out'), assert_returncode=None)
+      # TODO: add assert_returncode=NON_ZERO when node switches its default
```

Comment:
it's default what?

Reply:
node will currently return 0 for unhandled promise rejection, but the plan on changing that apparanetly.

Reply:
Sounds good, a comment maybe?

Reply:
Updated comment